packages/api-server/src/app.ts

```diff
@@ -2,16 +2,7 @@ import Fastify, { FastifyInstance } from 'fastify'
 
 export const createApp = (): FastifyInstance => {
   const app = Fastify({
-    logger: {
-      prettyPrint: process.env.NODE_ENV === 'development' && {
-        colorize: true,
-        ignore: 'hostname,pid,responseTime,reqId,req,res',
-        levelFirst: true,
-        messageFormat:
-          '{reqId} {msg} {res.statusCode} {req.method} {responseTime}',
-        translateTime: true,
-      },
-    },
+    logger: true,
```

Comment:
This'll have merge conflicts with our other change, just a headsup.

Reply:
Yes, I figured it would but needed a default there to get logging to test.